howtos/about_appliance_id.rst

```diff
@@ -0,0 +1,90 @@
+.. _aboutapplianceid:
+
+About Appliance ID
+==================
+
+During the initial :ref:`webui_setup` stage of a Rockstor install,
+an **Appliance ID** is automatically generated.
+This ID, intended as a Universal Unique Identifier (UUID),
+is an exact copy of the host system's, read motherboard's, response to:
+
+.. code-block:: console
+
+    cat /sys/class/dmi/id/product_uuid
+
+.. note::
+
+    With the majority of systems,
+    this is a genuinely unique reference; and no intervention of any sort is required.
+
+The implied unique-ness is assumed by multiple current Rockstor subsystems, e.g.
+when :ref:`add_appliance` as part of setting up a Share :ref:`sharereplication`.
+And as part of configuring a system to use the :ref:`stable_channel` updates.
+
+.. warning::
+
+    WARNING: some systems, via lax firmware implementations, return a NON-Unique / fake product_uuid.
+
+.. _knownfakeapplianceids:
+
+Fake Appliance ID examples
+--------------------------
+
+The following are examples on non-unique Appliance IDs,
+all recognised by the latest version of Rockstor.
+
+- **03000200-0400-0500-0006-000700080009** -
+  ASRock N3700-ITX, ASRock C2550D4I - Rockstor devs.
+  ASRock J3455 ITX - Thanks to forum member adworacz.
+- **00020003-0004-0005-0006-000700080009** -
+  ZOTAC 880G-ITX (880GITX-A-E) - Thanks to forum member mmmdonuts.
+- **5c4606fa-192f-453a-b299-7b088c63bb9b** -
+  GIADA N70E-DR - Thanks to forum member hammerite.
+- **31393138-3538-5a43-3135-353130323750** -
+  HP / HPE ProLiant MicroServer Gen8 - Thanks to David via support email.
```

Comment:
"HP/HPE ProLiant MicroServer Gen8 - Thanks to David via support email."

Just removing some spaces for readibility.